packages/config/README.md

```diff
@@ -64,6 +66,23 @@ const { config: webpackConfig, plugins } = config({
 |[useCloud](#use-cloud)|`boolean`|Toggle to use old fallback to cloud.redhat.com paths instead of console.redhat.com.|
 |[target](#target)|`string`|Override `env` and `useCloud` to use a custom URI.|
 
+
+#### experimentalTemplate
```

Comment:
Maybe we could change name of this option, I think it's not very clear.

examples:
`loadChromeTemplate`
`useChromeTemplate`
`loadChromeHtml`
etc.

Reply:
This option will eventually go away and will become part of the proxy.

Reply:
but I like `useChromeTemplate` better.